pyvista/core/pyvista_ndarray.py

```diff
@@ -111,7 +118,7 @@ def __setitem__(self, key: int | NumpyArray[int], value) -> None:
         if dataset is not None and dataset.Get():
             dataset.Get().Modified()
 
-    def __array_wrap__(self, out_arr, context=None, return_scalar: bool = False):
+    def __array_wrap__(self: pyvista_ndarray, out_arr, context=None, return_scalar: bool = False):  # noqa: ANN001, ANN204
```

Comment:
Could you explain us why we don't define the type of `out_arr` ?

Reply:
My IDE (PyCharm) gives this type hint by default:
<img width="627" alt="array_wrap" src="https://github.com/user-attachments/assets/44db9da4-e463-4087-8524-40958e537fbe">

It looked a bit too complex to reproduce, and I'm not sure these annotations would be much benefit in terms of external API type hints for users.

Reply:
Got it. Thanks!